CHANGELOG.md

```diff
@@ -12,6 +18,8 @@
 - Added an option when constructing the `RepositoryApiClient` to specify the requested scope(s) for the access token.
 - Added both `exportDocumentAsStream` and `exportDocumentWithAuditReasonAsStream` API methods in the `EntriesClientImpl` object along with the `EntriesClient` interface
 
+#### Note: accidently published version 2.0.1 so there is no difference between version 2.0.1 and 2.0.0
```

Comment:
Probably not necessary to mention this. I think if there are no changelog mentioned for a version, like 2.0.1, then it can be assumed there are no changes that impact users.